### PR TITLE
Added a bit more info to Github Actions page

### PR DESCRIPTION
**Pull request by walkerlj0**

### CODE REVIEW

**Comment by jackyzha0** [comment]
hi @walkerlj0, can you remove `node_modules` and reopen this PR?

**Comment by walkerlj0** [comment]
Gah sorry! That's going to be in all of em. I'll update my .gitignore

File: pages/deployment/github-actions.mdx [hunk 1]
```diff
@@ -4,18 +4,24 @@ import APIWarning from '../../components/ApiWarning.mdx'
 # Deploying with Github Actions
 <APIWarning/>
 
+[This Github action](https://github.com/marketplace/actions/publish-to-distributed-press) will zip your site together in a single file and use the Distributed Press to create and make updates to the content published on your web server every time it is triggered (this particular project is triggered when you push to the `main` branch of the Github Repo the Github Action is created in.
```
Comment (jackyzha0) [inline pages/deployment/github-actions.mdx:7]:
```suggestion
[This Github action](https://github.com/marketplace/actions/publish-to-distributed-press) will tar your site together into a single file and use the Distributed Press API to create and make updates to the content published on your web server every time it is triggered. This particular project is triggered when you push to the `main` branch of the Github Repo the Github Action is created in.
```

File: pages/deployment/github-actions.mdx [hunk 2]
```diff
@@ -76,3 +82,15 @@ jobs:
...
 This will build and publish your site to the Distributed Press
 instance hosted at `https://your-hosted.dp.instance`
+
+The  `with`: section of the Github Action has a some fields which are 
+required, such as `refresh_token` and `site_url` 
+(if you are hosting an HTTP site), since they are unique to your site. 
+The other fields that you want to specify can be also be added to your 
+Github action to customize your site deployment. 
+
+Each of the `deploy_http:`,  `deploy_hyper:`, and `deploy_ipfs` can be true or false,
```
Comment (jackyzha0) [inline pages/deployment/github-actions.mdx:92]:
```suggestion
Each of the `deploy_http:`,  `deploy_hyper:`, and `deploy_ipfs:` can be true or false, 
```

**Review by jackyzha0 — COMMENTED** [review]
A few nits, should be good after addressing

**Comment by walkerlj0** [comment]
Thanks for the edits, all committed @jackyzha0

**Review by jackyzha0 — APPROVED** [review]

**Pull request merged**